packaging/CPackConfig.cmake

```diff
@@ -14,3 +14,7 @@ if (CPACK_GENERATOR STREQUAL "External")
     set(CPACK_EXTERNAL_PACKAGE_SCRIPT "${CPACK_DEBIAN_UPLOAD_PPA_SCRIPT}" )
   endif ()
 endif()
+
+if(CPACK_SOURCE_GENERATOR OR DEB_SOURCEPKG)
+  list(APPEND CPACK_INSTALL_SCRIPTS "${CPACK_SOURCE_INSTALL_SCRIPT}")
```

Comment:
I think the CPACK_DEBIAN_UPLOAD_PPA_SCRIPT schould be responsible for writing this file. This way we can make sure that the version displayed in Mixxx marches the version of the package.

Reply:
This script is needed for both a plain source archive (TGZ generator) and source DEB. So I don't think it should be called from CPACK_DEBIAN_UPLOAD_PPA_SCRIPT. But now that I think about it, this probably needs to come first in the CPACK_INSTALL_SCRIPTS list order.

Reply:
There is no source deb generator.